lia-core/src/main/java/com/lithium/community/android/callback/Callback.java

```diff
@@ -0,0 +1,10 @@
+package com.lithium.community.android.callback;
+
+public interface Callback<R, T> {
```

Comment:
Just to call out using `T` for both `failure` and `abort` implies that they can never have different types. Shouldn't callback methods start with `on?

Reply:
starting with 'on'? does it make any difference?, Other review changes done, will update.

Reply:
I do not mind 😄 just asking.

Reply:
seriously, I want to know would it change the meaning?

Reply:
No, it won't, it is just an Android convention.